jqassistant.properties

```diff
@@ -0,0 +1,21 @@
+category=External Analyzers
+description=Analyze issues reported by jQAssistant.
+homepageUrl=https://github.com/jqassistant-contrib/sonar-jqassistant-plugin
+archivedVersions=1.7.0
+publicVersions=1.7.1
+ 
+defaults.mavenGroupId=org.jqassistant.contrib.sonarqube
+defaults.mavenArtifactId=sonar-jqassistant-plugin
+
+1.7.1.description=Reduced size of plugin JAR file
+1.7.1.date=2019-11-10
+1.7.1.sqVersions=[7.9,LATEST]
+1.7.1.changelogUrl=https://github.com/jqassistant-contrib/sonar-jqassistant-plugin/blob/master/release-notes.adoc#171
+1.7.1.downloadUrl=https://search.maven.org/remotecontent?filepath=org/jqassistant/contrib/sonarqube/sonar-jqassistant-plugin/1.7.1/sonar-jqassistant-plugin-1.7.1.jar
+
+1.7.0.description=Support for SonarQube 7.9 LTS and Maven multi-module projects
```

Comment:
There's almost no point in including this. We keep archived version data so the Marketplace can recognize when upgrades are available. Since no one has installed this version through the Mktplce, you've got a vanishingly small upgrade audience here. But this is up to you.